syntax = "proto3";
option optimize_for = LITE_RUNTIME;

package Traceability;

//ipfs表

message IpfsTable{
    string ipfskey         = 1; //key值唯一
    string orgNo           = 2;
    string productBatchNo  = 3;
    string productCode     = 4;
    string submitTime      = 5;
}

message ProductTable{
    string orgNo                    = 1;
    string productBatchNo           = 2;
    string filesHash                = 3;
    string fileType                 = 4;
    string address                  = 5;
    string lat                      = 6;
    string lng                      = 7;
    string createTime               = 8;
    string updateTime               = 9;
    string pinfoRecordId            = 10;
    string paccount                 = 11;
    string puserId                  = 12;
    string proleName                = 13;
    string premark                  = 14;
    string proleId                  = 15;
    string puserName                = 16;
}